### PR TITLE
[python3] migrate `ip_in_ip_tunnel_test.py` to python3 

### PR DESCRIPTION
**Pull request by zjswhhh**
### Description of PR

Summary:
Fixes # (issue)

Following instructions to migrate ptf test script `ip_in_ip_tunnel_test.py` to python3.

Sign-off: Jing Zhang zhangjing@microsoft.com 

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### How did you do it?
1. use `2to3` to update `ip_in_ip_tunnel_test.py`  script. 
2. add `is_python3==True` argument to ptf_runner called in dual_tor_utils.py.

#### How did you verify/test it?
1. dualtor/test_orchagent_standby_tor_downstream.py tests 
```
dualtor/test_orchagent_standby_tor_downstream.py::test_standby_tor_downstream_loopback_route_readded[ipv4]   PASSED
dualtor/test_orchagent_standby_tor_downstream.py::test_standby_tor_downstream_loopback_route_readded[ipv6]   PASSED 
```

### CODE REVIEW

File: ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py [hunk 1]
```diff
@@ -24,8 +24,8 @@
 PACKET_NUM_FOR_NEGATIVE_CHECK = 100
 
 DIFF = 0.25 # The valid range for balance check
-SRC_IP_RANGE = [unicode('8.0.0.0'), unicode('8.255.255.255')]
-SRC_IPV6_RANGE = [unicode('20D0:A800:0:00::'), unicode('20D0:FFFF:0:00::FFFF')]
+SRC_IP_RANGE = [str('8.0.0.0'), str('8.255.255.255')]
```
Comment (lolyu) [inline ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py:27]: Are literal strings Unicode by default in Python3?

File: ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ def setUp(self):
         self.standby_tor_ip = self.test_params['standby_tor_ip']
         self.ptf_portchannel_indices = self.test_params['ptf_portchannel_indices']
         self.indice_to_portchannel = {}
-        for port_channel, indices in self.ptf_portchannel_indices.items():
+        for port_channel, indices in list(self.ptf_portchannel_indices.items()):
```
Comment (lolyu) [inline ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py:54]: There is no need to make the iterable(`self.ptf_portchannel_indices.items()`) to be a list to make the loop work.

**Review by lolyu — COMMENTED** [review]
Looks like lots of code corrections from the migration script `2to3` are NOOPs.

File: ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py (hunk 1 above)
Comment (zjswhhh) [reply ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py:27]: Yes. 

Reference: https://docs.python.org/3/howto/unicode.html#the-string-type

File: ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py (hunk 2 above)
Comment (zjswhhh) [reply ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py]: Updated accordingly.

File: ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py (hunk 1 above)
Comment (lolyu) [reply ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py:27]: You could simply put like:
```
SRC_IPV6_RANGE = ['20D0:A800:0:00::', '20D0:FFFF:0:00::FFFF']
```

Comment (zjswhhh) [reply ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py]: updated accordingly.

**Review by lolyu — APPROVED** [review]

**Pull request merged**